src/Compilers/CSharp/Portable/Errors/MessageID.cs

```diff
@@ -216,6 +216,7 @@ internal enum MessageID
         IDS_FeatureVarianceSafetyForStaticInterfaceMembers = MessageBase + 12791,
         IDS_FeatureConstantInterpolatedStrings = MessageBase + 12792,
         IDS_FeatureMixedDeclarationsAndExpressionsInDeconstruction = MessageBase + 12793,
+        IDS_FeatureCallerArgumentExpression = MessageBase + 12794,
```

Comment:
>IDS_FeatureCallerArgumentExpression [](start = 8, length = 35)

It looks like we are missing the corresponding string in resourses

Reply:
Done.